tests/test_audio.py

```diff
@@ -398,6 +399,14 @@ def test_audio_Track_lyricStreams(track):
     assert not track.lyricStreams()
 
 
+@pytest.mark.authenticated
+def test_audio_Track_sonicAdventure(music):
```

Comment:
Do these tests work on the bootstrap test server? It only has one track and we disable all the analysis tasks (although it looks like we forgot to disable sonic analysis).

https://github.com/pkkid/python-plexapi/blob/b51aa1924b5b28ba61aefbb4162aff6e6864561b/tools/plex-bootstraptest.py#L502-L503

Also needs a Plex Pass account.

```suggestion
def test_audio_Track_sonicAdventure(account_plexpass, music):
```

Reply:
it might not, I changed my bootstrap server by adding a few tracks from my collection to test these features

Reply:
So this does not work on the bootstrap server with a single track, or even two tracks.

The tests fail at `assert len(adventure)` because there are no computed tracks. I think we just check if it is a list `assert isinstance(adventure, list)` instead.

Reply:
> So this does not work on the bootstrap server with a single track, or even two tracks.

would return list of 2 tracks, same as the args if not adventure between them.

Reply:
I tried this on my own server and it returns a blank list. The start and end are not included.

Reply:
okay, I would be wrong on this then, however the tests should be okay regardless